tools/cmake_call.sh

```diff
@@ -36,7 +36,7 @@ ${CMAKE_BIN} \
   -D NLOPT_SWIG=OFF \
   -D NLOPT_TESTS=OFF \
   ${CMAKE_ADD_AR} ${CMAKE_ADD_RANLIB} ../nlopt-src
-make -j${NCORES}
+make -j${NCORES:-1}
```

Comment:
Can you move the fixing logic higher up?  The script starts with

```
: ${R_HOME=$(R RHOME)}
RSCRIPT_BIN=${R_HOME}/bin/Rscript
NCORES=`${RSCRIPT_BIN} -e "cat(min(2, parallel::detectCores(logical = FALSE)))"`
```

Why does that not work in your settings?

Reply:
Ah good point. It seems to return `NA` on WebAssembly.